### DISCUSSION TITLE
Diff latest data with previous version?

### DISCUSSION THREAD

**Discussion by paambaati** [githubocto/flat-action#70] [Q&A]
What's the best approach to diff data between 2 versions? I'd like to build workflows that compares data across versions and take some action.

**Comment by Wattenberger** [comment]
You could create a postprocessing script that fetches the committed file contents (via the GitHub API) and compares it to the new file contents. The filename is passed as the first argument (`Deno.args[0]`), which can be used to get both of those paths.
https://github.com/githubocto/flat#postprocessing

Alternatively, you could also create another step in your workflow that looks at the last commit's diff and works from there.

**Comment by indifferentghost** [comment]
This is something I'm interested in to. I was thinking about doing it via the git diff, but all I need is to seed the changes into a in-memory db
